lib/scrabble/scrabble-class.rb

```diff
@@ -0,0 +1,61 @@
+module Scrabble
+  class Scrabble
+    SCORES = {
+      1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+      2 => ["d", "g"],
+      3 => ["b", "c", "m", "p"],
+      4 => ["f", "h", "v", "w", "y"],
+      5 => ["k"],
+      8 =>["j", "x"],
+      10 => ["q", "z"]
+    }
+
+    def self.score(word)
+      # Edge case - no input, less than 8 letters
+      if word == "" || word == nil || word.length > 7
+        return "ERROR"
+      end
+
+      # Starts word_score at zero
+      word_score = 0
+      word.downcase!
+
+      # Count points letter by letter
+      word.each_char do |char|
+        # Edge case - no numbers/spaces/weird things in your inputed word
+        if (char >= "a") && (char <= "z")
+          # Add the points associated with the letter "char" to the word_score
+          SCORES.select do |key, value|
+            if value.include?(char)
+              word_score += key
+            end
+          end
+        else
+          return "ERROR"
+        end
+      end
+
+      return word_score
+    end
+
+    def self.highest_score_from(array_of_words)
+      highest_score = 0
+      best_word = nil
+
+      array_of_words.each do |current_word|
+        current_word_score = score(current_word)
```

Comment:
Since `score` is a class method, you can clarify that you're using the class method by calling 
```
Scrabble.score(current_word)
```